src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: 7* $day-cell + 6*$gap;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-cell;
+    height: $day-cell;
+    border: 1px solid black;
+    background-color: #eee;
+    transition-duration: 0.5s;

```

Comment:
by default transition applies to all element styles. You need to specify what exactly you would like to have transition by adding `transition-property`